### PR TITLE
Distributing non-docs landing page links

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -144,26 +144,104 @@ module.exports = {
...
+        },
+        {
+          items: [
+            {
+              label: "Use Cases",
+              href: "https://temporal.io/use-cases",
+            },
+            {
+              label: "Case Studies",
+              href: "http://localhost:3000/blog/tags/case-study",
```
Comment (djmagee) [inline docusaurus.config.js:200]:
```suggestion
              href: "https://docs.temporal.io/blog/tags/case-study/",
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -144,26 +144,104 @@ module.exports = {
...
+            {
+              label: "Terms of Service",
+              href: "https://docs.temporal.io/pdf/temporal-tos-2021-07-24.pdf",
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "Join The Cloud Waitlist",
```
Comment (djmagee) [inline docusaurus.config.js:231]:
```suggestion
              label: "Join the Cloud Waitlist",
```
Although you might want to stet, at least for now, if you want to match the home page.

File: docusaurus.config.js [hunk 3]
```diff
@@ -144,26 +144,104 @@ module.exports = {
...
+              href: "https://us17.list-manage.com/survey?u=2334a0f23e55fd1840613755d&id=f1895b6f4a",
+            },
+            {
+              label: "Subscribe To The Newsletter",
```
Comment (djmagee) [inline docusaurus.config.js:235]:
```suggestion
              label: "Subscribe to the Newsletter",
```

File: docusaurus.config.js [hunk 4]
```diff
@@ -144,26 +144,104 @@ module.exports = {
...
+            {
+              label: "Blog",
+              to: "/blog",
+            },
+          ],
+        },
+        {
+          items: [
+            {
+              label: "About the docs",
```
Comment (djmagee) [inline docusaurus.config.js:211]:
```suggestion
              label: "About the Docs",
```
Although my edit doesn't match our style, it does match the rest of the footer (and the footer on the home page).

**Review by djmagee — APPROVED** [review]

File: docusaurus.config.js (hunk 1 above)
Comment (flossypurse) [reply docusaurus.config.js:200]: Whoops nice catch ;)

**Pull request merged**